lib/package-readme-view.js

```diff
@@ -45,13 +45,42 @@ function sanitize (html) {
     checkbox.setAttribute('disabled', true)
   }
 
+  let path = require('path')
+  let urlProtocolRegex = new RegExp("^(?:[a-z]+:)?\/\/", "i")
```

Comment:
Can't this be a simple constructed without the `RegExp` constructor, as in: `/^(?:[a-z]+:)?\/\//i`?  Then it can be inlined on L57.

Reply:
Done in latest commit